File: chapter03/hello2omp.c
#include <stdio.h>
#include  "omp.h"
int main(void)
{
	int id;

	#pragma omp parallel num_threads(4) private(id)
	{
	int id = omp_get_thread_num();
	printf("Hello world %d\n", id);
	}
return 0;
}

File: chapter03/hello2omp.f90
       program hello2omp
	include "omp_lib.h"
	integer :: id
	!$omp parallel num_threads(17) private(id)
	  id = omp_get_thread_num() 
          print *, "Hello world", id 
	!$omp end parallel
       end program hello2omp
